tests/chart_tests/test_pgbouncer.py

```diff
@@ -0,0 +1,47 @@
+import base64
+
+from tests.chart_tests.helm_template_generator import render_chart
+
+
+class TestPgbouncerSecret:
+    def test_pgbouncer_secret_defaults(self):
+        """Test pgbouncer secret custom server_idle_timeout"""
+        docs = render_chart(
+            values={
+                "airflow": {
+                    "pgbouncer": {
+                        "enabled": True,
+                    },
+                },
+            },
+        )
+        doc = [
+            doc
+            for doc in docs
+            if doc["metadata"]["name"] == "release-name-pgbouncer-config"
+        ][0]
```

Comment:
```suggestion
        docs = render_chart(
            values={
                "airflow": {
                    "pgbouncer": {
                        "enabled": True,
                    },
                },
            },
            show_only=["templates/secrets/pgbouncer-config-secret.yaml"],
        )[0]
```

Intentional not to use `show_only`?

Reply:
@jedcunningham do you know how to use `--show-only` with a subchart? I'd love to know.

Reply:
Really good point, I've edited my suggestion 👍.

Reply:
I *swear* I had a way to do this in the past. Maybe I was dreaming 😂

Reply:
it only works if the chart is expanded. when i do `helm dep update` i get a tgz but it does not expand. i was thinking of modifying the test runner to expand the tgz file. do you have that feature already implemented some place?

Reply:
That's really odd, mine isn't expanded and it works. Not that I know of?

```
$ git clean -fxd
$ helm dependency update
...
Saving 1 charts
Downloading airflow from repo https://airflow.apache.org
Deleting outdated charts
$ ll charts
total 296
-rw-r--r--  1 jedc  staff   147K Jul  6 17:32 airflow-1.6.0.tgz
$ helm template --show-only charts/airflow/templates/secrets/pgbouncer-config-secret.yaml foo .
---
# Source: airflow/charts/airflow/templates/secrets/pgbouncer-config-secret.yaml
# Licensed to the Apache Software Foundation (ASF) under one
# or more contributor license agreements.  See the NOTICE file
# distributed with this work for additional information
# regarding copyright ownership.  The ASF licenses this file
# to you under the Apache License, Version 2.0 (the
# "License"); you may not use this file except in compliance
# with the License.  You may obtain a copy of the License at
#
#   http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing,
# software distributed under the License is distributed on an
# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
# KIND, either express or implied.  See the License for the
# specific language governing permissions and limitations
# under the License.

################################
## Pgbouncer Config Secret
#################################
```

Reply:
wow, yeah, that works for me too. SWEET! now i need to fix his in pytest.

Reply:
d'oh, I had addd this because I thought it would be a good idea.
```
if not Path(i).exists():
    raise FileNotFoundError(f"ERROR: {i} not found")
```
Once again, it would be great if we just turned this thing into a pytest module so we could pool our expertise on it. Thanks for pushing back on my suboptimal code!